doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='create_archetype_project']
+= Creating a project using archetypes
+
+Archetypes are projects that are installed in Apache Maven repositories and contain a specific template structure. You can also generate parameterized versions of the project templates using archetypes. You can use archetypes to create a project in {CENTRAL}. When you create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.
```

Comment:
Remove this:
You can use archetypes to create a project in {CENTRAL}.

Change this:
When you use an archetype to create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.

Note that my version might not be 100 percent correct - I'm not familiar with the feature.